shared/test/context_messenger.js

```diff
@@ -97,7 +97,7 @@ describe ("Messaging Test Suite", function() {
 
 describe('Privly.message.adapter.*', function () {
 
-  it('can remove listeners', function (done) {
+  it('can add and remove listeners', function (done) {
```

Comment:
Good clarification, but it would be better to have this in a separate commit since it is not related to adding event emitter test coverage.